File: src/main/resources/VM/python/file_run_28/Main.py
import sys

def get_index_of_two_elm_equal_target(nums, target):
    # Code you here
    return []

if __name__ == "__main__":
    if len(sys.argv) < 3:
        sys.exit(1)
    num_strs = sys.argv[1].split(",")
    nums = list(map(int, num_strs))
    target = int(sys.argv[2])
    output = get_index_of_two_elm_equal_target(nums, target)
    print(output)

File: src/main/resources/VM/rust/file_run_28/Main.rs
use std::env;
use std::collections::HashMap;

fn main() {
    let args: Vec<String> = env::args().collect();

    if args.len() != 3 {
        return;
    }

    let nums_str = &args[1];
    let target: i32 = args[2].parse().expect("Target must be an integer");

    let nums: Vec<i32> = nums_str
        .split(',')
        .map(|s| s.trim().parse().expect("Invalid number in nums"))
        .collect();

    let result = two_sum(nums, target);
    println!("{:?}", result);
}

fn two_sum(nums: Vec<i32>, target: i32) -> Vec<i32> {
    // code cua ban o day
    vec![]
}